.gitignore

```diff
@@ -1,2 +1,3 @@
 # Ignore binary
 /secrethub
+/.idea
```

Comment:
I recommend setting up a [global gitignore](https://stackoverflow.com/questions/7335420/global-git-ignore) that contains entries to ignore files for the IDE you're using. By doing so, you don't have to change the `.gitignore` of all repositories you're working on.

Reply:
Thanks for the tip.
I removed the line for consistency.